utils/notification.ts

```diff
@@ -120,11 +120,12 @@ export const infoMessage = (
   })
 }
 
-export const successMessage = message =>
+export const successMessage = (message: string, extraConfig?: { footer?: Ref<Omit<NotificationAction, 'url'> | undefined> }) =>
```

Comment:
if you could make a `NotificationFooter` type

```suggestion
export const successMessage = (message: string, extraConfig?: { footer?: NotificationFooter | undefined> }) =>
```

Reply:
updated